### ISSUE TITLE
Request to update for version 1.32.72.0.0

### ISSUE THREAD

**Issue by Link8232** [ronbrogan/H2Randomizer#4]
Thought I'd look at it myself but I'm not a coder in the slightest. My guess was the offset to find the level name from memory needed to be updated but I couldn't figure out how to find the needed info. Thanks

Edit: Learned you can downpatch to make it work, but I'm still curious how to get it working for the current version.

**Comment by RevanXVIV** [comment]
Yeah I'd like to see a modern release version too so I don't have to downpatch.
